src/gen_smtp_client.erl

```diff
@@ -526,7 +526,11 @@ connect(Host, Options) ->
 		_ ->
 			25
 	end,
-	case socket:connect(Proto, Host, Port, SockOpts, 5000) of
+	Timeout = case proplists:get_value(timeout, Options) of
```

Comment:
@mworrell but now someone may still pass `{timeout, blablabla}`, so, not a good argument I think.